fedimint-core/src/core.rs

```diff
@@ -72,10 +72,10 @@ impl OperationId {
         Self(encodable.consensus_hash::<sha256::Hash>().to_byte_array())
     }
 
-    pub fn fmt_short(&self) -> OperationIdShortFmt {
+    pub fn fmt_short(&'_ self) -> OperationIdShortFmt<'_> {
```

Comment:
Not a big deal but I think to get clippy to pass you only need `OperationIdShortFmt<'_>`, and not `&'_ self`